desktop/cmp/grid/Grid.js

```diff
@@ -217,9 +218,10 @@ export class Grid extends Component {
             const displaySpec = action.getDisplaySpec(params);
             if (displaySpec.hidden) return;
 
-            let subMenu;
+            let childItems;
             if (!isEmpty(displaySpec.items)) {
-                subMenu = this.buildMenuItems(displaySpec.items, record, selectedRecords);
+                const menu = new StoreContextMenu({items: displaySpec.items, gridModel: this.gridModel});
```

Comment:
Doing this so we fully support all of the special string tokens for the sub menu items